guides/common/modules/con_restricting-hosts-access-to-content.adoc

```diff
@@ -0,0 +1,65 @@
++
+For more information about Content Views, see xref:Managing_Content_Views_{context}[].
++
+For more information about lifecycle environments, see xref:Creating_an_Application_Life_Cycle_{context}[].
+
+Content overrides::
+By default, content hosted by {Project} can be either enabled or disabled.
+In custom products, repositories are always disabled by default, while Red{nbsp}Hat products can be either enabled or disabled by default depending on the specific repository.
+Enabling a repository gives the host access to the repository packages or other content, allowing hosts to download and install the available content.
++
+If a repository is disabled, the host is not able to access the repository content.
+A content override provides you with the option to override the default enablement value of either *Enabled* or *Disabled* for any repository.
+You can add content overrides to hosts or activation keys.
++
+For more information about adding content overrides to hosts, see {ManagingHostsDocURL}Enabling_and_Disabling_Repositories_on_Hosts_managing-hosts[Enabling and Disabling Repositories on Hosts] in _{ManagingHostsDocTitle}_.
++
+For more information about adding content overrides to activation keys, see xref:enabling-and-disabling-repositories-on-activation-key_{context}[].
+
+Composite Content Views::
+You can use composite Content Views to combine and give hosts access to the content from multiple Content Views.
+For more information about composite Content Views, see xref:Creating_a_Composite_Content_View_{context}[].
+
+Architecture and OS version restrictions::
+In custom products, you can set restrictions on the architecture and OS versions for `yum` repositories on which the product will be available.
+For example, if you restrict a custom repository to *{EL} 9*, it is only available on hosts running {EL} 9.
+Architecture and OS version restrictions hold the highest priority among all other strategies.
+They cannot be overridden or invalidated by content overrides, changes to Content Views, or changes to lifecycle environments.
+For this reason, it is recommended to consider the other strategies mentioned before using archictecture or OS version restrictions.
+Red{nbsp}Hat repositories set architecture and OS version restrictions automatically.
+
+Release version::
+Certain Red{nbsp}Hat repositories, such as the {EL} dot release repositories, include a "release version" in their repository metadata.
+The release version is then compared with the release version specified in the system purpose attributes of the host.
+Access to content may be limited or restricted based on this comparison.
+For more information about setting system purpose attributes,see {ManagingHostsDocURL}Creating_a_Host_managing-hosts[Creating a Host in {ProjectName}] in _{ManagingHostsDocTitle}_.
+
+.Putting everything together
+A particular package or repository is available to a host only if all of the following are true:
```

Comment:
I had that wording in the original draft I wrote because some of the conditions apply to entire repositories (e.g. arch and OS version restrictions) and some (such as content view filters) apply to specific packages within repositories.

Reply:
Looking again, content view filters are the only item that may apply to either a repository or an individual package. All other items in this list apply to repositories, if that helps.